DiveLane/Presentation Layer/Controllers/WalletCreation/WalletCreationViewController.swift

```diff
@@ -300,6 +300,19 @@ class WalletCreationViewController: UIViewController {
         }
     }
     
+    func checkPasswordsMatch() -> Bool {
+        return !((!(passwordTextField.text?.isEmpty ?? true) ||
```

Comment:
as I mention in https://github.com/matterinc/DiveLane/pull/30#discussion_r220602963 better to prettify it with exttension: https://stackoverflow.com/a/52139426

Reply:
Looks very convenient! Let me fix it after merge in order to avoid duplicating the code.

Reply:
Ok! https://github.com/matterinc/DiveLane/issues/36